chutney/ui/src/app/modules/dataset/components/dataset-list/dataset-list.component.html

```diff
@@ -84,6 +90,61 @@
                 }
               </td>
               <td class="pt-2"><span>{{dataset.description}}</span></td>
+              <td class="pt-2">
+                  <ng-template #scenarioUsage>
+                      <div class="text-start">
+                          @for (scenarioUsage of dataset.scenarioUsage; track $index) {
+                              <span>Scenario <strong>{{scenarioUsage}}</strong></span><br/>
+                          }
+                      </div>
+                  </ng-template>
+                  @if (dataset.scenarioUsage.length > 0) {
+                      <div class="btn btn-sm position-relative"
+                           [ngbTooltip]="scenarioUsage" placement="top">
+                          <span class="fa-solid fa-2xl fa fa-film text-info"></span>
+                          <span class="position-absolute top-0 start-75 translate-middle badge border border-info rounded-circle text-info">
+                          {{ dataset.scenarioUsage.length }}
+                        </span>
+                      </div>
+                  }
+                  <ng-template #campaignUsage>
+                      <div class="text-start">
+                          @for (campaignUsage of dataset.campaignUsage; track campaignUsage) {
+                              <span>Campaign <strong>{{campaignUsage}}</strong></span><br/>
+                          }
+                      </div>
+                  </ng-template>
+                  @if (dataset.campaignUsage.length > 0) {
+                      <div class="btn btn-sm position-relative"
+                           [ngbTooltip]="campaignUsage" placement="top">
+                          <span class="fa-solid fa-2xl fa fa-clock text-success"></span>
+                          <span class="position-absolute top-0 start-75 translate-middle badge border border-success rounded-circle text-success">
+                          {{ dataset.campaignUsage.length }}
+                        </span>
+                      </div>
+                  }
+                  <ng-template #scenarioInCampaignUsage class="mw-100">
+                      <div class="text-start d-flex flex-column">
+                          @for (scenarioInCampaignUsage of Object.keys(dataset.scenarioInCampaignUsage); track scenarioInCampaignUsage) {
+                              <span class="pb-1 pt-1">Campaign <strong>{{scenarioInCampaignUsage}}</strong> :</span>
+                              <div class="d-flex flex-column">
+                                  @for (scenario of dataset.scenarioInCampaignUsage[scenarioInCampaignUsage]; track $index) {
+                                    <span class="ps-4">Scenario <strong>{{scenario}}</strong></span>
+                                  }
+                              </div>
+                          }
+                      </div>
+                  </ng-template>
+                  @if (Object.keys(dataset.scenarioInCampaignUsage).length > 0) {
```

Comment:
![image](https://github.com/user-attachments/assets/9dbc468c-025f-43d8-b3fc-4beb062630f8)

Reply:
`Object` used in html is undefined(because it isn't known as component property).
May be you have to declare readonly `readonly Object = Object;` in ts